### PR TITLE
Glamorous implementation

### PR DESCRIPTION
**Pull request by anicarrr**
Here it goes!

### CODE REVIEW

**Comment by pomber** [comment]
Awesome!

File: web/package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "react-router-dom": "^4.1.1"
   },
   "devDependencies": {
+    "glamor": "^2.20.25",
+    "glamorous": "^3.23.2",
```
Comment (pomber) [inline web/package.json]: Shouldn't these be `dependencies` instead of `devDependencies`?

File: web/src/Form.js [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
   render() {
     return (
-      <div className="form" style={this.formStyle}>
-        <Spinner style={this.spinnerStyle} />
+      <FormContainer
+        height={this.props.height}
+        width={this.props.width}
+        radius={this.props.radius}
+        alpha={this.props.alpha}
+      >
```
Comment (pomber) [inline web/src/Form.js]: I would add `const {height, width, radius, alpha, loading} = this.props;` to avoid using `this.props` every time.

**Review by pomber — CHANGES_REQUESTED** [review]

File: web/src/Form.js (hunk 2 above)
Comment (anicarrr) [reply web/src/Form.js]: I was about to do that!

Something that makes noise for me is the fact of transferring props for styling and business logic in the same component. Maybe the Getter approach for dynamic styling was more cleaner.

Comment (pomber) [reply web/src/Form.js]: I agree, looks better with the `get formStyle()`

Comment (anicarrr) [reply web/src/Form.js]: Maybe we can use glamorous for static styling and for the dynamic ones use Getters.

**Comment by pomber** [comment]
Add back the getters for dynamic styles and fix the dependencies in package.json and I'll merge this PR. We can continue with the rest of the files with another PR. Thanks!

**Comment by anicarrr** [comment]
Now it's a good for merging ;)

I'll will continue with the rest on another PR.

**Review by pomber — APPROVED** [review]

**Comment by pomber** [comment]
Niiiice

**Pull request merged**